Whitnessme codesandbox

whitnessme opened the pull request.

whitnessme committed:
Create codesandboxFAQs.md

whitnessme committed:
Rename codesandboxFAQs.md to codesandbox-FAQs.md

whitnessme committed:
Update codesandbox-FAQs.md

flow-state-15 commented:
@whitnessme Add a line after Table of Contents to remind them to create their own sandbox accounts first, so they don't lose their work. 

We can add a link to this doc under 'Resources' heading [here](https://github.com/flow-state-15/mod-5-extra-resources/tree/main) 

Add a section for opening the built in browser view in a new window so they can use React devTools

flow-state-15 reviewed (changes requested):
check my comments

whitnessme committed:
Update README.md

whitnessme committed:
adding screenshots for codesandbox faqs

whitnessme committed:
Add files via upload

whitnessme committed:
Update codesandbox-FAQs.md

added step to make sure students are logged in; added react devtools sections with images

whitnessme commented:
Review requested of the added material. _Note:_ the table of contents links won't work until its on the main branch. 
- [x] Add a line after Table of Contents to remind them to create their own sandbox accounts first, so they don't lose their work.
- [x] We can add a link to this doc under 'Resources' heading [here](https://github.com/flow-state-15/mod-5-extra-resources/tree/main)
- [x] Add a section for opening the built in browser view in a new window so they can use React devTools

whitnessme committed:
Add files via upload

add screenshot photo for codesandbox instructions

whitnessme committed:
Update codesandbox-FAQs.md

updated save to the fork method and added the save as possible bug to the saving section.
